tests/test-cases/Group8-vSphere-Integration/8-1-GuestTools.robot

```diff
@@ -14,32 +14,34 @@ Verify container VM guest IP is reported
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${id}=  Run And Return Rc And Output  docker ${params} run -d busybox /bin/top
+    ${name}=  Generate Random String  15
+    ${rc}  ${id}=  Run And Return Rc And Output  docker ${params} run --name ${name} -d busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.ip ${name}-*
     Should Be Equal As Integers  ${rc}  0
-    Should Not Contain  ${id}  Error
-    Run  govc vm.ip ${id}
 
 Stop container VM using guest shutdown
     ${rc}=  Run And Return Rc  docker ${params} pull busybox
     Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${id}=  Run And Return Rc And Output  docker ${params} run -d busybox /bin/top
+    ${name}=  Generate Random String  15
+    ${rc}  ${id}=  Run And Return Rc And Output  docker ${params} run --name ${name} -d busybox /bin/top
     Should Be Equal As Integers  ${rc}  0
-    Run  govc vm.ip ${id}
-    ${rc}=  Run And Return Rc  govc vm.power -s ${id}
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.power -s ${name}-*
```

Comment:
does this check the resulting power state or just whether the shutdown was delivered correctly?